spec/support/now.rb

```diff
@@ -0,0 +1,17 @@
+require 'time'
+
+module Support
+  module Now
+    def self.included(base)
+      base.let(:now) { Time.now }
+      base.before { allow(Time).to receive(:now).and_return(now) }
+    end
+  end
+end
+
+# https://stackoverflow.com/questions/44259104/rubyzip-undefined-method-to-binary-dos-time-for
+require 'zip'
+
+Zip::DOSTime.instance_eval do
+  def now; Zip::DOSTime.new(); end # ugh.
```

Comment:
What is `ugh` for?

Reply:
it's a horrible monkey patch that is needed because rubyzip does something wild that breaks creating the zip file when you stub out `Time.now` in tests. there are a couple open issues on the repo about this